website/frontend/src/components/pages/Mailbox/children/Message/Message.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from 'react';
+import { web3FromSource } from '@polkadot/extension-dapp';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { useApi } from 'hooks/useApi';
+import { RootState } from 'store/reducers';
+import MessageIcon from 'assets/images/message.svg';
+import ClaimIcon from 'assets/images/claim.svg';
+import { AddAlert } from 'store/actions/actions';
+import { getPreformattedText } from 'helpers';
+import { EventTypes } from 'types/alerts';
+import { MessageType } from '../../types';
+import styles from './Message.module.scss';
+
+type Props = {
+  elem: MessageType;
+};
+
+export const Message: FC<Props> = ({ elem }) => {
```

Comment:
```suggestion
const Message: = ({ message }) => {
```

- `export` to bottom
- drop `FC` type
- `message` is more declarative name